docs/content/tutorial/disaster-recovery.md

```diff
@@ -112,7 +112,7 @@ spec:
       options:
       - --type=time
       - --target="2021-06-09 14:15:11-04"
-  image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-13.4-1
+  image: {{< param imageCrunchyPostgresCentos >}}
   postgresVersion: 13
```

Comment:
Do we want to have a param for the postgres version too? Since that will need to match the image tag :question:

Reply:
sure will do